rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -892,22 +892,47 @@ void opcode_0051(const ScriptArguments& args) {
 }
 
 /**
-    @brief  %5d% = create_player %1o% at %2d% %3d% %4d%
+    @brief LINE
```

Comment:
Consider giving this a new name, or avoid implementing this in the first place.
If the command is not used, there is no point in implementing (or even stubbing) it either *imo*.

Reply:
This one is unused.